vars/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-rhel7stig_min_ansible_version: 2.4
+rhel7stig_min_ansible_version: 2.5
```

Comment:
Is there a problem with the role and Ansible 2.4?

Reply:
This was just to keep up with the latest so my users were not running 2.4 instead of 2.5 when 2.6 is already out.

Reply:
Will you also address the deprecation warnings such as "Using tests as filters is deprecated"

Reply:
@dirtyharrycallahan see #144. Are there outlier cases that I missed?

Reply:
I only use the role on a system w/o internet access so I won't know until next week when I perform my transfers.

Reply:
We don't need to change this in the role at this time. If it is to make sure local users are not running older versions of Ansible I think that is better handled as a site specific thing. i.e. site policy about versions of software packages or overriding this variable locally. 

We would update this value to a newer version if we started using features, modules, etc that only exist in the newer version of Ansible. Hence it is the minimum required Ansible version.